src/runtime/native/runtime.ml

```diff
@@ -10,16 +10,33 @@ let default_bisect_file = ref "bisect"
 
 let default_bisect_silent = ref "bisect.log"
 
+let sigterm_enable = ref false
+
+let sigterm_ignore_overriden_warned = ref false
+
+let sigterm_ignore_handle_warned = ref false
```

Comment:
I've added these warnings that are emitted if an existing signal handler is overwritten. 
To not output them in each file visited, I track their emission in these references.

Reply:
I would lean towards omitting these warnings, unless you already foresee that you might have difficulty with silent replacement of SIGTERM handlers in your own usage.

A user that is using `--bisect-sigterm` or `BISECT_SIGTERM` has already explicitly opted into having Bisect install a SIGTERM handler. If we ever have a real user that has a complex enough assortment of binaries in testing that they would benefit from this warning, we can add the warnings then.

It's also likely that rather than warning, we would want to "chain" to the previous signal behavior &mdash; not exit the process if the signal was ignored, or call the other function if there was already a handler.

However, a binary under test that installs its own signal handlers outside the Bisect runtime is likely not to interact well with `--bisect-sigterm` anyway, since it will tend to install its own signal handler *after* `--bisect-sigterm`. That means that both the warnings and attempts to chain signal handlers are likely not to work anyway, since `--bisect-sigterm` will be processed first, and then the binary will clobber `--bisect-sigterm` behavior.

Reply:
Makes sense to me. It can be useful when you're instrumenting a big application and you don't actually know whether it installs it's own signal handlers or not. 

Warnings removed in 04c164d.